dvc/repo/get.py

```diff
@@ -30,6 +31,11 @@ def get(url, path, out=None, rev=None):
             #
             # Also, we can't use theoretical "move" link type here, because
             # the same cache file might be used a few times in a directory.
+
+            # Note: we need to replace state, because in case of getting DVC
+            # dependency on CIFS or NFS filesystems, sqlite-based state
+            # will be unable to obtain lock
+            repo.state = StateBase()
```

Comment:
You separated comment from its code by inserting it here.

Reply:
I am not sure what do you mean? It should be below `repo.state = StateBase()`?

Reply:
The comment "Also, we can't use theoretical "move" link type here ... " is about the code "repo.config.set(..." and you added your lines inbetween.

Reply:
Ahh, sorry.